logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/OutgoingMessageRecipientsRetriever.kt

```diff
@@ -0,0 +1,91 @@
+package com.wire.kalium.logic.feature.message
+
+import com.wire.android.core.exception.Failure
+import com.wire.android.core.functional.Either
+import com.wire.android.core.functional.suspending
+import com.wire.android.feature.contact.DetailedContact
+import com.wire.android.feature.conversation.content.MessageRepository
+import com.wire.android.feature.conversation.data.ConversationRepository
```

Comment:
We are pulling another `ConversationRepository`? Where does this come from?

Reply:
This whole class was actually not supposed to be in this PR. This is on higher-level on top of fetching pre-keys and recipients of a conversation.

The pre-keys part is on its way, and then I'll add another PR with this.